app/scripts/controllers/mmi-controller.js

```diff
@@ -569,7 +569,7 @@ export default class MMIController extends EventEmitter {
         const mmiDashboardData = await this.handleMmiDashboardData();
         const cookieSetUrls =
           this.mmiConfigurationController.store.mmiConfiguration?.portfolio
-            ?.cookieSetUrls;
+            ?.cookieSetUrls || [];
```

Comment:
For some unknown reason, sometimes this could be undefined. The `setDashboardCookie` needs `cookieSetUrls` to be an array so that's why we provide an empty array as a default value